backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/CassandraTypesCreator.java

```diff
@@ -22,10 +22,17 @@
 import org.apache.james.backends.cassandra.components.CassandraModule;
 import org.apache.james.backends.cassandra.components.CassandraType;
 
+import com.datastax.driver.core.KeyspaceMetadata;
 import com.datastax.driver.core.Session;
 import com.google.common.collect.ImmutableList;
 
 public class CassandraTypesCreator {
+    public enum InitialisationStatus {
```

Comment:
How is it different from the one of the tables?

Reply:
It is not, it's just Types' specific